README.md

```diff
@@ -21,210 +22,110 @@ Nmstate is aimed to satisfy enterprise needs to manage host networking through
-```
+```python
+import libnmstate
 
-Alternatively, install Nmstate system-wide:
-```shell
-pip uninstall -y nmstate; pip install .
+# Specify a Linux bridge (created if it does not exist).
+state = {'interfaces': [{'name': 'br0', 'type': 'linux-bridge', 'state': 'up'}]}
+libnmstate.apply(state)
 ```
 
-### Container Image
-
-Nmstate also provides a container image based on CentOS 7 to try it:
-
 ```shell
-CONTAINER_ID=$(sudo docker run --privileged -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro nmstate/centos7-nmstate)
-sudo docker exec -ti "${CONTAINER_ID}" /bin/bash
-# now play with nmstatectl in the container
-nmstatectl show
-# remove the container at the end
-sudo docker stop "${CONTAINER_ID}"
-sudo docker rm "${CONTAINER_ID}"
+# use yaml or json formats
+nmstatectl set desired-state.yml
+nmstatectl set desired-state.json
 ```
 
-## Basic Operations
+Edit the current state(python/shell):
+```python
+import libnmstate
 
-Show current state:
-```shell
-nmstatectl show
-```
+state = libnmstate.show()
+eth0_state = next(ifstate for ifstate in state['interfaces'] if ifstate['name'] == 'eth0')
 
-Change to desired state:
-```shell
-nmstatectl set desired-state.yml
-nmstatectl set desired-state.json
+# take eth0 down
+eth0_state['state'] = 'down'
+libnmstate.apply(state)
 ```
 
-Edit current state of eth3 in a text editor:
 ```shell
+# open current state in a text editor, change and save to apply
 nmstatectl edit eth3
 ```
 
+## Contact
 
-`nmstatectl` will also read from stdin when no file is specified:
-
+*Nmstate* uses the [NetworkManager mailing
+list](https://mail.gnome.org/mailman/listinfo/networkmanager-list)
+([Archives](https://mail.gnome.org/archives/networkmanager-list/)) for
+discussions. Emails about nmstate should be tagged with `[nmstate]` in the
+subject header to ease filtering.
 
-```shell
-nmstatectl set < desired-state.yml
-```
+Development planning (sprints and progress reporting) happens in
+([Jira](https://nmstate.atlassian.net)). Access requires login.
 
-Desired/Current state example (JSON):
-```shell
-{
-    "interfaces": [
-        {
-            "description": "Production Network",
-            "ethernet": {
-                "auto-negotiation": true,
-                "duplex": "full",
-                "speed": 1000
-            },
-            "ipv4": {
-                "address": [
-                    {
-                        "ip": "192.0.2.142",
-                        "prefix-length": 24
-                    }
-                ],
-                "enabled": true
-            },
-            "mtu": 1500,
-            "name": "eth3",
-            "state": "up",
-            "type": "ethernet"
-        }
-    ]
-}
-```
+There is also `#nmstate` on
+[Freenode IRC](https://freenode.net/kb/answer/chat).
 
-See `nmstatectl --help` for additional command-line options.
+## Installation
 
-## Supported Interfaces:
-- bond
-- dummy
-- ethernet
-- ovs-bridge
+Please see the [install](README.install.md) instructions.
```

Comment:
Can we move the installation section to the top, something like:

```markdown
For Fedora 29+, `sudo dnf install nmstate -y`.
Others distribution, please see the [install](README.install.md) instructions.
```

Reply:
IMHO we do not need to mention the Fedora version, since 29 is the oldest one that is currently supported. Therefore we could just remove it to make it more timeless.

Reply:
I think that @cathay4t intention by the `+` is to say that it is available from that version on.
I guess we can fit somewhere in between by stating `the latest supported Fedora release`.

Reply:
Done